src/graphtypes/simplegraphs/simplegraph.jl

```diff
@@ -139,6 +139,54 @@ function add_edge!(g::SimpleGraph, e::SimpleGraphEdge)
     return inserted
 end
 
+function add_edges!(g::SimpleGraph, es)
+    T = eltype(g)
+    nvnew = nvold = nv(g)
+    touched = falses(nvold)
+    neold = ne(g)
+    for e in es
+        s, d = T.(Tuple(e))
+        if s < 1 || d < 1
+            error("Nonpositive vertex number")
```

Comment:
A little more feedback would be helpful such as: error("Nonpositive vertex number $s, $d")